config/default/jenkins-infra.yaml

```diff
@@ -183,23 +183,6 @@ jenkins:
                           }
                         }
                       }
-                  - script: >
-                      multibranchPipelineJob('docker-mirrorbits') {
-                        displayName "Mirrorbits Docker Build"
```

Comment:
Is it wanted?

Reply:
the mirror bits config was duplicated, I've been meaning to sort that out for a little while